app/views/users/index.html.erb

```diff
@@ -1,2 +1,21 @@
-<h1>All Users</h1>
-<p>Here is a list of all users</p>
+<ul>
+  <% @users.each do |user| %>
+    <li>
+      <div class="user-div">
+        <div class="photo">
+          <% if user.photo.present? %>
+            <%= image_tag user.photo, alt: "User Photo", class: "user-photo" %>
+          <% else %>
+            <span>No photo</span>
+          <% end %>
+        </div>
+        <%= link_to user_path(user) do %>
+          <div class="user-info">
+            <span class="user-name"><%= user.name %></span>
+            <span class="post-count">Number of posts: <%= user.posts.count %></span>
+          </div>
```

Comment:
- [ ] **[OPTIONAL]** It would have been nice to make this into a partial to clean up the views and also easily import this into other view files